ci-automation/azure-sig.sh

```diff
@@ -0,0 +1,368 @@
+#!/bin/bash
+
+set -euo pipefail
+
+[[ -n ${DEBUG:-} ]] && set -o xtrace
+
+# Make sure Azure credentials are given without printing them.
+tracestate="$(shopt -po xtrace || true)"
+set +o xtrace
+_="
+${AZURE_CLIENT_ID}
+${AZURE_CLIENT_SECRET}
+"
```

Comment:
If we don't require them as env vars, we can remove this

Reply:
If you want parallel usage through manual invocation, you can move this to the `else` case of the block below.